AnaProd/anaTupleProducer.py

```diff
@@ -273,7 +278,8 @@ def createAnatuple(inFile, outDir, config, sample_name, anaCache, snapshotOption
             for puIDbranch in puIDbranches:
                 if puIDbranch in dfw.df.GetColumnNames():
                     new_branch_name= puIDbranch.strip("_tmp")
-                    dfw.DefineAndAppend(new_branch_name, f"{puIDbranch}[ExtraJet_B1]")
+                    #dfw.DefineAndAppend(new_branch_name, f"{puIDbranch}[ExtraJet_B1]")
```

Comment:
don't comment. should be inside `if config["GLOBAL"]["storeExtraJets"]`

Reply:
ok